sonar-plugin-api/src/main/java/org/sonar/api/utils/log/DefaultProfiler.java

```diff
@@ -94,24 +94,31 @@ public Profiler startInfo(String message) {
 
   @Override
   public Profiler stopTrace() {
-    return doStopWithoutMessage(LoggerLevel.TRACE);
+    return doStopWithoutMessage(LoggerLevel.TRACE, " (done)");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal " (done)" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)